react/features/prejoin/components/dialogs/DialInDialog.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import React from 'react';
+import ActionButton from '../buttons/ActionButton';
+import { translate } from '../../../base/i18n';
+import { Icon, IconArrowLeft } from '../../../base/icons';
+import Label from '../Label';
+import { getCountryCodeFromPhone } from '../../utils';
+
+type Props = {
+
+    /**
+     * The number to call in order to join the conference.
+     */
+    number: string,
+
+    /**
+     * Handler used when clicking the back button.
+     */
+    onBack: Function,
+
+    /**
+     * Click handler for the text button.
+     */
+    onTextButtonClick: Function,
+
+    /**
+     * Click handler for primary button.
+     */
+    onPrimaryButtonClick: Function,
+
+    /**
+     * Click handler for the small additional text.
+     */
+    onSmallTextClick: Function,
+
+    /**
+     * The pin of the conference.
+     */
+    pin: string,
+
+    /**
+     * Used for translation.
+     */
+    t: Function,
+};
+
+/**
+ * This component displays the dialog whith all the information
+ * to join a meeting by calling it.
+ *
+ * @param {Props} props - The props of the component.
+ * @returns {React$Element}
+ */
+function DialinDialog(props: Props) {
+    const {
+        number,
+        onBack,
+        onPrimaryButtonClick,
+        onSmallTextClick,
+        onTextButtonClick,
+        pin,
+        t
+    } = props;
+    const flagClassName = `prejoin-dialog-flag iti-flag ${getCountryCodeFromPhone(
+        number,
+    )}`;
+
+    return (
+        <div className = 'prejoin-dialog-dialin'>
+            <div className = 'prejoin-dialog-dialin-header'>
+                <Icon
+                    className = 'prejoin-dialog-icon prejoin-dialog-dialin-icon'
+                    onClick = { onBack }
+                    size = { 24 }
+                    src = { IconArrowLeft } />
+                <div className = 'prejoin-dialog-title'>
+                    {t('prejoin.dialInMeeting')}
+                </div>
+            </div>
+            <Label number = { 1 }>{t('prejoin.dialInPin')}</Label>
```

Comment:
Didn't the linter warn for missing space after `{`?

Reply:
The linter said it was ok, but fixed it.